refactron/rag/indexer.py

```
@@ -195,39 +214,55 @@ def _index_file(self, file_path: Path, summarize: bool = False) -> List[CodeChun
         if not chunks:
             return
 
-        # Prepare data for ChromaDB
+        if self.mode == "keyword":
+            # Keyword mode: append to JSON storage
+            current_data = []
+            if self.chunk_storage.exists():
+                try:
+                    with open(self.chunk_storage, "r", encoding="utf-8") as f:
+                        current_data = json.load(f)
+                except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)